README.md

```diff
@@ -1,8 +1,12 @@
-[![Go Report Card](https://goreportcard.com/badge/github.com/istio/istio)](https://goreportcard.com/report/github.com/istio/istio)
-[![GoDoc](https://godoc.org/istio.io/istio?status.svg)](https://godoc.org/istio.io/istio)
+<a href="https://istio.io/">
+    <img src="logo/istio-bluelogo-whitebackground-framed.svg" alt="istio logo" title="istio" align="right" width="200" height="200" />
```

Comment:
You should use `istio-bluelogo-nobackground-unframed.svg` from official assets mentioned by @sdake

Reply:
Hello @shamsher31 yes I thought of using the direct URL(https://istio.io/latest/img/istio-bluelogo-whitebackground-framed.svg) but I think it is better to have the image inside the project because:
* user won't load the image from network
* it is easier to distinguish the logo between different versions
* it is more reliable if we store it in the same project

so, I have downloaded the mentioned images from the official sites and kept in inside istio/logo file.

Reply:
Keeping it in the `logo` folder is standard and also followed by [Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/logo/logo.svg).